src/components/filterCheckbox/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+
+import { chakra } from '@chakra-ui/react';
+
+import Flex from '../flex';
+import Checkbox from '../checkbox';
+import Box from '../box';
+
+interface Item {
+  value: string | number;
+  label: string;
+}
+
+export interface Props {
```

Comment:
I think we could type that pretty safely with generics. Let's have a quick call about this.